tests/xtro-sharpie/tvOS-NetworkExtension.ignore

```diff
@@ -0,0 +1,8 @@
+## Native code has NWPathStatus in NetworkExtension and nw_path_status_t in Network (with the same elements).
+## NetworkExtension.NWPathStatus is the oldest, but NetworkExtension isn't present on all the platforms Network
+## is, which means we can't use the same managed enum for both cases (the native enums aren't entirely identical
+## either: one is pointer sized while the other is 32-bit always).
+!duplicate-type-name! NWPathStatus enum exists as both NetworkExtension.NWPathStatus and Network.NWPathStatus
+
+!unknown-field! NEAppProxyErrorDomain bound
```

Comment:
encountered a weird error when running xtro tests, but doesn't make sense because the entirety of the class is not available on tvOS? added to the ignore for the time being..